Adding beginner friendly precisions (#234)
scalar-labs/scalardb: docs/getting-started-with-scalardb.md

First of all, you need to define how the data will be organized (a.k.a database schema) in the application with Scalar DB database schema.
Here is a database schema for the sample application. For the supported data types, please see [this doc](schema.md) for more details.

> You can create a JSON file `emoney-transaction.json` by the JSON object below.

Reviewer: ```suggestion
You can create a JSON file `emoney-storage.json` with the JSON below.
```
Since it's for storage.

> In this tutorial we use a json file named `emoney-storage.json` filled with the json object below . 

Reviewer 2: Nit:
```suggestion
In this tutorial we use a json file named `emoney-storage.json` filled with the json object below. 
```

```

Now we can run the application.

> If you run those command in the getting-started repository do not forget to create an empty file named `setting.gradle`.

Reviewer 2: I found this issue last Friday, and I added `setting.gradle` in #233. So I think we can remove this line. Thanks.

Reviewer 3: Yes, Above sentence is not needed because `setting.gradle` available in the repository
It is working fine.

> If you run those command in the getting-started repository do not forget to create an empty file named `setting.gradle`.

Reviewer: ```suggestion
```

$ ../../gradlew run --args="-mode storage -action pay -amount 100 -to merchant1 -from user1"
```

## Set up database schema for transaction

> To apply transaction, we can just add a key `transaction` and value as `true` in Scalar DB scheme.

Reviewer: ```suggestion
To use transaction, we can just add a key `transaction` and value as `true` in the Scalar DB scheme we used.
```

> You can modify the JSON file `emoney-storage.json` by the json object bellow.

Reviewer: ```suggestion
You can create the JSON file `emoney-transaction.json` with the JSON below.
```
We are using `emoney-transaction.json` below so I think we should mention the name.

## Set up database schema with transaction

To apply transaction, we can just add a key `transaction` and value as `true` in Scalar DB scheme.

> You can modify the json file `emoney-storage.json` by the json object bellow.

Reviewer 3: @feeblefakie san,
Sorry to disturb you.
Can we modify it like  `You can create a JSON file emoney-transaction.json by the JSON object below`.

> You can modify the json file `emoney-storage.json` by the json object bellow.

Author: At this line we already created the JSON file. We are doing the second test using transacion as true, so i suppose that modify is ok here.
